hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/handle/stack/SavepointStackImplTest.java

```diff
@@ -46,14 +46,14 @@ class SavepointStackImplTest extends StateTestBase {
     private static final Configuration BASE_CONFIGURATION = HederaTestConfigBuilder.createConfig();
     private static final String FOOD_SERVICE = "FOOD_SERVICE";
 
-    private static final Map<String, String> BASE_DATA = Map.of(
+    private static final Map<String, String> BASE_DATA = new HashMap<>(Map.of(
```

Comment:
I don't get why this _base_ data map (static!) needs to be mutable.  Seems like it _shouldn't_ be mutable as it will be used for _all_ tests here.  `setup` should be making copies for each test run, no?

Reply:
Ah great point...made a non-`static` member in case more than one test mutates this map in the future. 😌